src/bin/pgcopydb/pgsql.c

```diff
@@ -2024,80 +2023,6 @@ pgsql_handle_notifications(PGSQL *pgsql)
 }
 
 
-/*
- * hostname_from_uri parses a PostgreSQL connection string URI and returns
- * whether the URL was successfully parsed.
- */
-bool
-hostname_from_uri(const char *pguri,
-				  char *hostname, int maxHostLength, int *port)
```

Comment:
Thanks for the clean-up, I think we should consider doing that in a separate PR though.

Reply:
The function wasn't entirely in different context from the PR's(connection strings, URI, passwords etc.), so removed it in this PR itself. Can do a separate cleanup if that's better?

Reply:
Yes let's do a separate PR for cleanup here. Makes it easier to review commits in the git history when we have to dive into advanced issues later.